src/gretel_trainer/relational/artifacts.py

```diff
@@ -37,7 +38,12 @@ def upload_transforms_outputs_archive(self, project: Project, path: str) -> None
         existing = self.transforms_outputs_archive
         self.transforms_outputs_archive = self._upload_file(project, path, existing)
 
-    def _upload_file(self, project: Project, path: str, existing: Optional[str]) -> str:
+    def _upload_file(
+        self, project: Project, path: str, existing: Optional[str]
+    ) -> Optional[str]:
+        if self.hybrid:
+            return None
```

Comment:
nit: this makes sense given the description in the PR, about polluting the hybrid storage buckets, but might be worth adding some details about that here as it's not obvious by just looking at this code why we skip the upload for hybrid

Reply:
Good call 👍